### ISSUE TITLE
Unable to parse the latest dron ontology using Pronto

### ISSUE THREAD

**Issue by Zethson** [mcwdsi/dron#37]
```
  File ~/miniconda3/envs/lamindb/lib/python3.11/site-packages/pronto/ontology.py:283 in __init__
    cls(self).parse_from(_handle)  # type: ignore

  File ~/miniconda3/envs/lamindb/lib/python3.11/site-packages/pronto/parsers/rdfxml.py:84 in parse_from
    tree: etree.ElementTree = etree.parse(handle)

  File ~/miniconda3/envs/lamindb/lib/python3.11/xml/etree/ElementTree.py:1218 in parse
    tree.parse(source, parser)

  File ~/miniconda3/envs/lamindb/lib/python3.11/xml/etree/ElementTree.py:580 in parse
    self._root = parser._parse_whole(source)

  File <string>
ParseError: no element found: line 9693809, column 11
```

https://github.com/althonos/pronto is the standard to load owl/ontologies in Python. Some older versions like 2024-03-02 still seem to work for me.

**Comment by hoganwr** [comment]
We have been changing how DrOn is built.

To confirm, which release of DrOn?  We just made two recently: v2024-07-29
and v2024-08-05

And which file specifically?  Assuming it's OWL and the OWL file available
at the PURL, then line 9693809 is respectively:

v2024-07-29:
                            </owl:Restriction>

v2024-08-05:
                <owl:someValuesFrom>

Nothing obvious there, so more details on the release you're parsing and
the particular file (a DrOn release technically comprises 12 files, four
files in 3 formats each - OWL, OBO, and JSON).

**Comment by Zethson** [comment]
Thanks! Sorry for the lack of details.

I used http://purl.obolibrary.org/obo/dron/releases/2024-07-29/dron.owl .

**Comment by hoganwr** [comment]
Thanks!

So far I am unable to reproduce the issue.

On the interactive shell, I did:
from pronto import Ontology
dron = Ontology("
http://purl.obolibrary.org/obo/dron/releases/2024-07-29/dron.owl")

That threw up a bunch of SyntaxWarning, but I didn't get the error message
you did.

I also tried putting those two lines of code in load_dron.py, and ran:
python3 load_dron.py

Same result.

Using python 3.12, and latest pronto (I did pipenv install pronto)
